highfidelity/hifi
# Tablet ui entity statistics

Test plan:
- Open entities statistics
- Make sure statistics updated once per second
- Check realtime statistics checkbox
- Open entities statistics
- Make sure statistics updated in real-time
- Make sure colors are rotating
- Check servers statistics details changed pressing Less/Most/More/Least buttons
